### PR TITLE
CDK: Review overview/quick start

### PR DESCRIPTION
**Pull request by kmurphypolygon**
- Updates per review.
- Scanned and corrected the quick start where it failed.
- Fixed up general language issues, style guide misses, failing links, etc.
- Restructure and emphasis on important things for UX.
- Housekeeping.

### CODE REVIEW

File: docs/cdk/getting-started/local-deployment.md [hunk 1]
```diff
@@ -1,43 +1,52 @@
...
 ```bash
 git clone https://github.com/0xPolygon/kurtosis-cdk.git
 cd kurtosis-cdk
 ```
 
-### Checking your environment
+### Check your environment
 
-Ensure Docker is running on your machine, then run the following command to confirm that all prerequisites are installed:
+Run the `tool_check.sh` script to confirm you have all the prerequisite software.
```
Comment (gsilva5) [inline docs/cdk/getting-started/local-deployment.md:43]: We chose to be more verbose in those guide to make it more conversational.  Let's revert back to the old language here as this is more terse than the previous version.

File: docs/cdk/overview.md [hunk 2]
```diff
@@ -2,26 +2,34 @@
...
 
-Polygon Chain Development Kit (CDK) is a modular, open source software toolkit for blockchain developers which supports the installation and configuration of a variety of chain architectures. Polygon CDK empowers developers to launch new L2 chains running Polygon zkEVM technology on Ethereum or, in the future, transition existing Layer 1 (L1) chains into ZK-EVM L2s. 
+Polygon Chain Development Kit (CDK) is a modular, open source, software toolkit for blockchain developers which supports the installation and configuration of a variety of chain architectures. 
+
+Polygon CDK empowers developers to launch new L2 chains running Polygon zkEVM technology on Ethereum or, with optional validium networks and, in the future, transition existing layer 1 (L1) chains into custom ZK-EVM L2s. 
 
 ## CDK features
 
-Polygon CDK provides the components necessary to build a layer 2 blockchain that is secure, scalable, and interoperable with other chains. Below are some of the key features of the CDK:
+Polygon CDK builds layer 2 blockchains that are secure, scalable, and interoperable with other chains. Below are some of the key features of CDK:
```
Comment (gsilva5) [inline docs/cdk/overview.md]: I like the old language here of "Polygon CDK provides the components necessary to ..."

Comment (kmurphypolygon) [reply docs/cdk/overview.md]: it's clunky

File: docs/cdk/getting-started/local-deployment.md (hunk 1 above)
Comment (kmurphypolygon) [reply docs/cdk/getting-started/local-deployment.md:43]: The thing is, you don't need Docker running to run this script, so it's incorrect to say to ensure it.

File: docs/cdk/overview.md (hunk 2 above)
Comment (kmurphypolygon) [reply docs/cdk/overview.md]: updated

**Review by EmpieichO — APPROVED** [review]

**Pull request merged**